### PR TITLE
feat: xlsx filter

### PR DESCRIPTION
**Pull request by degenaro**
## Types of changes

- [ ] Hot fix (emergency fix and release)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Documentation (change which affects the documentation site)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Release (`develop` -> `main`)

## Quality assurance (all should be covered).

- [x] My code follows the code style of this project.
- [x] Documentation for my change is up to date?
- [x] My PR meets testing requirements.
- [x] All new and existing tests passed.
- [x] All commits are signed-off.

## Summary

Support for optional filtering of xlsx by specified column name containing value "yes".

## Key links:

- [Sonar coverage](https://sonarcloud.io/dashboard?id=compliance-trestle)

# Before you merge

- Ensure it is a 'squash commit' if not a release.
- Ensure CI is currently passing
- Check sonar. If you are working for a fork a maintainer will reach out, if required.

### CODE REVIEW

File: tests/trestle/tasks/xlsx_to_oscal_profile_test.py [hunk 1]
```diff
@@ -79,6 +79,32 @@ def test_xlsx_execute(tmp_path):
         assert (result)
 
 
+@patch(target='uuid.uuid4', new=uuid_mock1)
+@patch(target='trestle.tasks.xlsx_to_oscal_profile.get_trestle_version', new=get_trestle_version_mock1)
+def test_xlsx_execute_filter(tmp_path):
+    """Test execute filter call."""
+    config = configparser.ConfigParser()
+    config_path = pathlib.Path('tests/data/tasks/xlsx/test-xlsx-to-oscal-profile.config')
+    config.read(config_path)
```
Comment (fsuits) [inline tests/trestle/tasks/xlsx_to_oscal_profile_test.py:88]: Might be good to define a const path for tasks in test_utils.py.  See the top of that file

File: trestle/tasks/xlsx_helper.py [hunk 2]
```diff
@@ -405,3 +444,5 @@ def report_issues(self) -> None:
             logger.info(f'rows missing parameters: {self.rows_missing_parameters}')
         if len(self.rows_missing_parameters_values) > 0:
             logger.info(f'rows missing parameters values: {self.rows_missing_parameters_values}')
+        if len(self.rows_filtered) > 0:
+            logger.info(f'rows filtered: {self.rows_filtered}')
```
Comment (fsuits) [inline trestle/tasks/xlsx_helper.py:448]: can just be 
if self.rows_filtered:

**Review by fsuits — APPROVED** [review]
LGTM minor comments

File: tests/trestle/tasks/xlsx_to_oscal_profile_test.py (hunk 1 above)
Comment (degenaro) [reply tests/trestle/tasks/xlsx_to_oscal_profile_test.py:88]: Fixed, kinda. The file path reference is specific to this test script. Same for xlsx_to_oscal_cd_test. I'm not so sure moving to test_utils.py is the best option. However, I did make a constant variable for each at the respective tops.

File: trestle/tasks/xlsx_helper.py (hunk 2 above)
Comment (degenaro) [reply trestle/tasks/xlsx_helper.py]: I have a lot of these in the xlsx_ code: if len(x) > 0. All fixed.

**Pull request merged**